baccmod/base_acceptance_map_creator.py

```diff
@@ -845,10 +843,11 @@ def create_acceptance_map_cos_zenith_binned(self,
                 if self.use_mini_irf_computation:
                     evaluation_time, observation_time = get_time_mini_irf(obs, self.mini_irf_time_resolution)
 
-                    data_obs_all = np.zeros(tuple([len(evaluation_time), ] + list(ref_model.data.shape))) * ref_model.unit
+                    data_obs_all = []
```

Comment:
Why removing the pre allocation of the table ?
It add more manipulation of data with append and transformation to an array.